### ISSUE TITLE
Automation in Logic 

### ISSUE THREAD

**Issue by mk-conn** [the-synister/the-source#1]
First I have to say: Really cool plugin, thanx for sharing it! 

I just noticed that if I enable 'touch' automation mode and tweak e.g. the clipping parameter nothing happens. Not a big deal as one can find the clipping the in the automation parameter list, but it would be faster to draw some automation and edit afterwards (thats the way I work with other plugins though).

**Comment by juanecabellob** [comment]
Thank you for your issue! 
We'll have a look into it and write you back as soon as we figured it out!

**Comment by juanecabellob** [comment]
Does it happens when the Clipping effect is on? The parameter for that is Clipping Active, please check because I couldn't reproduce

**Comment by mk-conn** [comment]
yep, just recorded a screen session (without sound) where you can see it

[https://www.mediafire.com/?yg010ygfy7dfsa7](https://www.mediafire.com/?yg010ygfy7dfsa7)

**Comment by juanecabellob** [comment]
Thanks, that helps a lot! I'll let you know when we've fixed it.
